src/auth/auth.service.ts

```diff
@@ -51,11 +51,16 @@ export class AuthService {
         },
       });
       if (!user) {
-        return res.json({ error: 'Not user found with given email!' });
+        return res.json({
+          error: 'Nie znaleziono użytkownika o podanym e-mailu!',
+        });
+      }
+      if (!user.isActive) {
+        return res.json({ error: 'Your account is deactivated!' });
```

Comment:
Skoro resztę errors zwracasz po polsku, to ten też proponuję po polsku
```suggestion
        return res.json({ error: 'Twoje konto jest dezaktywowane' });
```